src/rtc_time.cpp

```diff
@@ -0,0 +1,90 @@
+/**
+ * @file
+ * @brief     CPP file for the RTCTime struct.
+ * @author    Jasper Smienk
+ * @license   MIT License
+ */
+
+#include "rtc_time.hpp"
+
+uint8_t RTCTime::getSeconds() {
+    return seconds;
+}
+
+uint8_t RTCTime::getMinutes() {
+    return minutes;
+}
+
+uint8_t RTCTime::getHours() {
+    return hours;
+}
+
+uint8_t RTCTime::getDayOfTheWeek() {
+    return dayOfTheWeek;
+}
+
+uint8_t RTCTime::getDayOfTheMonth() {
+    return dayOfTheMonth;
+}
+
+uint8_t RTCTime::getMonth() {
+    return month;
+}
+
+uint8_t RTCTime::getYear() {
+    return year;
+}
+
+
+
+void RTCTime::setSeconds(uint8_t seconds) {
+    this->seconds = seconds;
```

Comment:
There is no need for the this pointer here, you can just use `seconds = seconds`.
The same goes for the rest of the file.